app/styles/app.css

```diff
@@ -0,0 +1,3 @@
+.click-icon {
+  cursor: pointer;
+}
```

Comment:
TODO: Find why this doesn't work, and basically improve the feeling or plus/minus. cf. previous remark.